.github/workflows/ci-orchestrator.yml

```diff
@@ -19,19 +19,36 @@ jobs:
 
   ci_docker:
     needs: changes
-    if: ${{ needs.changes.outputs.docker == 'true' }}
-    uses: ./.github/workflows/python-build-and-push-docker-image.yml
+    uses: Energinet-DataHub/.github/.github/workflows/python-build-and-push-docker-image.yml@v12
+    with:
+      docker_changed: ${{ needs.changes.outputs.docker == 'true' }}
+      docker_changed_in_commit: ${{ needs.changes.outputs.docker_in_commit == 'true' }}
 
   create_release:
-    needs: [changes]
+    needs: changes
     if: ${{ needs.changes.outputs.telemetry == 'true' || needs.changes.outputs.spark-sql-migrations == 'true' }}
     uses: ./.github/workflows/create-release-tag.yml
     secrets: inherit
 
+  #
+  # Run Python  unit tests
+  #
+  python_ci_test:
+    needs: [ci_base, ci_docker]
+    if: always()
+    uses: Energinet-DataHub/.github/.github/workflows/python-ci.yml@v12
+    with:
+      operating_system: dh3-ubuntu-20.04-4core
+      path_static_checks: ./source
+      # documented here https://github.com/Energinet-DataHub/opengeh-migration/tree/main/source/MigrationTools#styling-and-formatting
+      ignore_errors_and_warning_flake8: E501,F401,E402,E203,W503
+      test_report_path: ./source
+      image_tag: ${{ needs.ci_docker.outputs.image_tag }}
+
   # Branch policy status check
   allow_merge_ci_orchestrator:
     runs-on: ubuntu-latest
-    needs: [ci_base, ci_docker, create_release]
+    needs: [ci_base, ci_docker, create_release, python_ci_test]
     if: |
       always()
```

Comment:
NIT: Isn't this kind of irrelevant / redundant? The job is already triggered by the completion of the other jobs?

Reply:
The jobs have to be completed successfully. This means, that if a job has been skipped, it will not be triggered. 

I experienced it with the new "ci_docker". It skips whenever it does not need to build a new docker image

Reply:
Alright (y) :D